bin/genesis

```diff
@@ -1550,6 +1555,30 @@ sub detect_bosh_version {
 	}
 }
 
+sub system_bosh {
+	my ($cmd) = join(" ", ($BOSH, @_));
```

Comment:
Perl automatically flattens lists, so `join(' ', ($scalar, @list)` is identical to `join(' ', $scalar, @list)`

You can remove the superfluous parentheses.